webrtc.html

```diff
@@ -3836,8 +3836,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
        value as if it had come from RFC 6464 and use that.  If the
        source was set from a CSRC, this will be the level value
        defined in [[!RFC6465]].  RFC 6464 and 6465 define the level as
-       a integral value from 0 to -127 representing the audio level in
-       decibels relative to the loudest signal that they system could
+       a integral value from 0 to 127 representing the audio level in
+       negative decibels relative to the loudest signal that they system could
```

Comment:
Is this term readily understood? We could add "Thus, 0 represents the loudest signal the system could possibly encode, and 127 represents silence."